aws-lc-rs/src/lib.rs

```diff
@@ -103,6 +103,7 @@ extern crate core;
 
 pub mod aead;
 pub mod agreement;
+mod buffer;
```

Comment:
minor nit: Noticed this when checking if `Buffer` was a new public interface, should we move `buffer` down to list of other non-public modules below